src/components/DeletablePill/README.md

```diff
@@ -0,0 +1,6 @@
+# DeletablePill
+
+When checked, clicking anywhere on a deletable pill fires an `onDelete` function.
```

Comment:
Not sure the deletable pill should always assume clicking on it deletes it. Especially if there's also the clear icon. Also, i find naming the x icon "clear" is sooooo confusing... i thought it was just a blank space like we have in tk-ui for the longest time